source/layouts/application_layout.html.erb

```diff
@@ -24,6 +24,14 @@
 <% end %>
 
 <% wrap_layout :core do %>
+  <% app_slug = current_page.path.match(/apps\/([^\/]+)\/(?:.+)/) %>
```

Comment:
Not quite - otherwise we end up with an extra (unwanted) breadcrumb, which links to `http://localhost:4567/apps/govuk-docker.html.html`:

<img width="941" alt="Screenshot 2021-05-06 at 07 59 18" src="https://user-images.githubusercontent.com/5111927/117254952-fe266080-ae40-11eb-902b-821c10d5f246.png">

Reply:
That worked 👍 added!